src/views/Login.jsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import AuthForm from '../components/AuthForm';
+import { signInUser, signUpUser } from '../services/users';
+
+export default function Login({ setUser }) {
```

Comment:
The name here suggests this component is only concerned with logging a user in, not signing them up too